framework/SwiftPlot/LineChart.swift

```diff
@@ -228,10 +228,10 @@ extension LineGraph{
             }
         }
 
-        var maximumXSecondary: Float = 0
-        var maximumYSecondary: Float = 0
-        var minimumXSecondary: Float = 0
-        var minimumYSecondary: Float = 0
+        var maximumXSecondary = T(0)
```

Comment:
@BradLarson I had to add those extra initializers in FloatConvertible.swift to do this.